content/en/get-started/deploy-your-app.md

```diff
@@ -10,50 +10,61 @@ tags: ["Deployment", "Quickstart"]
 
 ## Learning objectives
 
-In this guide, you build on what you learned in the  CD-as-a-Service [Quickstart]({{< ref "get-started/quickstart" >}}) by deploying your own app using the deployment configuration from the Quickstart.
+1. [Create a deployment config file](#create-a-deployment-config-file)
+2. [Deploy your app](#deploy-your-app)
+3. [Monitor your deployment](#monitor-your-deployment)
 
-1. [Create your deployment config file](#create-a-deployment-config-file)
-1. [Deploy your app](#deploy-your-app)
-
-
->If you prefer a web-based, interactive tutorial, see the CD-as-a-Service Console's [**Deploy Your Own App** tutorial](https://next.console.cloud.armory.io/getting-started).
 
 ## {{% heading "prereq" %}}
 
-* You have completed the CD-as-a-Service [Quickstart]({{< ref "get-started/quickstart" >}}).
+* You have the [Armory CLI installed]({{< ref "cli.md" >}}).
```

Comment:
The page should stand on its own, Lets have collapsible section on how to install CLI, and how to install RNA.

See example here - https://github.com/armory/docs/pull/2047/files#diff-0f0bc3c26ed0b12cf6ab4fb2ddeade37fae2b10eb328d3ecce75cea1da27c17eR61